mysite/mysite/settings.py

```diff
@@ -126,3 +126,9 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_HOST_USER = "neomaddy104@gmail.com"
+EMAIL_HOST_PASSWORD = "gkiizqlykfgoowqx"
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
```

Comment:
https://alicecampkin.medium.com/how-to-set-up-environment-variables-in-django-f3c4db78c55f